.circleci/workflows.yml

```diff
@@ -853,8 +853,12 @@ jobs:
           steps:
             - *skip
   deploy-to-private-gcr:
-    executor: ubuntu-machine-executor
+
+    executor: gcp-cli/google
     steps:
+      - run: *oidc-setup
+      - gcp-cli/setup:
```

Comment:
I don't think we need to add a `gcp-cli/setup` here, since https://circleci.com/developer/orbs/orb/circleci/gcp-gcr#commands-gcr-auth and https://circleci.com/developer/orbs/orb/circleci/gcp-gcr both support `use_oidc` directly. It's not immediately obvious to me which of the two should be updated (possibly both).

https://github.com/mozilla/bigquery-etl/blob/main/.circleci/workflows.yml#L868
https://github.com/mozilla/bigquery-etl/blob/main/.circleci/workflows.yml#L872

Side note: these orbs [might need updates](https://github.com/mozilla/bigquery-etl/blob/main/.circleci/workflows.yml#L5) to get the OIDC flags.

Reply:
updated GCP CLI, GCR, and Docker (since its closely related to the GCR one) here https://github.com/mozilla/bigquery-etl/pull/5898/commits/46d48cbc7b17d6670a946928fa37a1b08a043958

Reply:
My thought is that `gcp-cli/setup`  isn't necessary since you can just add `use_oidc` to one or both of the lines I linked above. I'm not super confident in that assertion but I think Mikael had used `gcp-cli/setup`  since we were executing arbitrary commands via `gcloud` whereas image building is done via standard upstream orbs.